test/multinetwork.jl

```diff
@@ -469,7 +469,6 @@ TESTLOG = Memento.getlogger(PowerModels)
         @test_throws(TESTLOG, ErrorException, PowerModels.correct_voltage_angle_differences!(mn_data))
         @test_throws(TESTLOG, ErrorException, PowerModels.correct_thermal_limits!(mn_data))
         @test_throws(TESTLOG, ErrorException, PowerModels.correct_branch_directions!(mn_data))
-        @test_throws(TESTLOG, ErrorException, PowerModels.check_branch_loops(mn_data))
```

Comment:
As mentioned before, I think it's best to keep the scope of `check_branch_loops` as is during the update.  In that case this test can be restored.